src/ps/api/client/mod.rs

```diff
@@ -1217,6 +1243,7 @@ pub mod tests {
     const TEST_ENVIRONMENT: Environment = Environment::NonProduction;
     const TEST_API_KEY: &str = env!("PENNSIEVE_API_KEY");
     const TEST_SECRET_KEY: &str = env!("PENNSIEVE_SECRET_KEY");
+    const TEST_API_ID: &str = env!("PENNSIEVE_API_ID");
```

Comment:
I don't love that we have this new `PENNSIEVE_API_ID` that the clients need to know about in order to authenticate. What would you think about exposing a new un-authenticated endpoint that serves up the Cognito client and pool IDs? Then the clients could hit that endpoint when they first start up and grab to the token pool client ID.

Otherwise we are going to have to configure these client IDs all over the place for the different environments. Would be much cleaner if the clients could figure this out themselves.

Reply:
Yeah I think could work 👍

Reply:
That shouldn't block merging this, but let's try and clean that up soon.

Reply:
Ticket here to track: https://app.clickup.com/t/rd87ae